src/afrolabs/components/kafka.clj

```diff
@@ -1410,7 +1506,18 @@
 
       IDeref
       (deref [_]
-        @has-caught-up-once
+        ;; wait, but with feedback every 5 seconds
+        (let [starting-millis (System/currentTimeMillis)]
+          (loop []
+            (let [caught-up? (deref has-caught-up-once
+                                    5000
+                                    :timeout)]
+              (when (= :timeout caught-up?)
+                (log/info (str "KTable '" consumer-group-id "' loading initial data. Waited "
+                               (long (/ (- (System/currentTimeMillis) starting-millis)
+                                        1000))
+                               " seconds so far."))
+                (recur)))))
```

Comment:
When a caller is waiting for the ktable to load (by using `deref`/`@`) we produce a log statement every 5 seconds so the waiting is not invisible.